.gitignore

```diff
@@ -4,3 +4,5 @@ npm-debug.log
 bower_components
 dist
 *.swp
+.cache
+.idea
```

Comment:
I got it for `.idea`, but why `.cache` ?

Reply:
That is the sass cache.